dandischema/models.py

```diff
@@ -958,7 +964,7 @@ class CommonModel(DandiBaseModel):
         None, readOnly=True, description="permalink to the item", nskey="schema"
     )
     repository: HttpUrl = Field(
-        "https://dandiarchive.org/",
+        DANDI_WEB_APP_URL,
```

Comment:
@satra -- I wonder if we should completely remove a default value here in the "generic" `Resource` data type -- why should it be dandi specific by default?

Reply:
the only reason is that the schema is quite dandi specific at the moment. btw this section says `DANDI_WEB_APP_URL`? can we call it `DANDI_INSTANCE_URL`.  i don't want this url to be `gui.dandiarchive.org`. this has to be the `dandiarchive.org` for the production instance.

Reply:
I don't care much about variable name, just was matching the one from terraform configuration, but can change tomorrow to the one you like

Reply:
ok, since we are still here -- I have renamed it into `DANDI_INSTANCE_URL` and `DANDI_ARCHIVE_PATTERN` (after searching for it in the organization -- not used) into `DANDI_INSTANCE_URL_PATTERN` for consistency.